src/QualificationRunner.Core/Domain/BuildingBlockRef.cs

```diff
@@ -0,0 +1,13 @@
+﻿using OSPSuite.Core.Domain;
+using OSPSuite.Core.Qualification;
+
+namespace QualificationRunner.Core.Domain
+{
+   public class BuildingBlockRef : IWithName, IReferencingProject

```

Comment:
This was moved by mistake. It did not belong in core as this is only read by the QualificationRunner